protocol/src/types/block.rs

```diff
@@ -359,6 +359,7 @@ pub struct Proof {
     pub number:     u64,
     #[cfg_attr(feature = "hex-serialize", serde(serialize_with = "serialize_uint"))]
     pub round:      u64,
+    #[serde(rename(serialize = "proposal_hash"))]
```

Comment:
Why rename only `serialize`?

Reply:
Maybe `derive(Deserialize)` on `Proof` should be removed? Or you should remove the `serialize` in `rename` so that serialization and deserialization matches.